src/content/docs/reference/recipe.mdx

```diff
@@ -49,6 +61,23 @@ The tag of the base image to build on. Used to select a version explicitly (`40`
 image-version: 40
 ```
 
+### `stages:`
+
+A list of [stages](/references/stages/) that is executed outside of the final image. This is useful for compiling programs from source without polluting the final bootable image.
+
+#### Example:
+
+```yaml
+stages:
+  - name: bluebuild
+    from: rust
```

Comment:
use a full container URL here please, to make it clearer what this means

Reply:
Good idea